frontend/src/app/components/Contact/Contact.tsx

```diff
@@ -0,0 +1,54 @@
+import styles from "@/app/components/Sidebar/sidebar.module.scss";
+import matter from "gray-matter";
+import ReactMarkdown from "react-markdown";
+import fs from "fs/promises";
+import path from "path";
+
+async function getContactMarkdown() {
+    try {
+        const filePath = path.join(process.cwd(), "content", "contact.md");
+
+        const raw = await fs.readFile(filePath, "utf8");
+
+        return matter(raw);
+
+    } catch (error) {
+        console.error("Error reading contact.md:", error);
+        return { data: {}, content: null };
+    }
+}
```

Comment:
In this function, Typescript threw type errors when I tried to build it in production. Also, the `cwd` path is different in development than in production, so there needs to be some change. I could make it work like this:

```suggestion
type ContactData = {
  title: string;
  email: string;
  address: string;
};

type ContactMarkdown = {
  data: ContactData | null;
  content: string | null;
};

async function getContactMarkdown(): Promise<ContactMarkdown> {
  try {
    const filePath = path.join(process.env.HOME || "", "content", "contact.md");

    const raw = await fs.readFile(filePath, "utf8");

    const parsed = matter(raw) as GrayMatterFile<string> & {
      data: Partial<ContactData>;
    };

    const data: ContactData = {
      title: parsed.data.title ?? "",
      email: parsed.data.email ?? "",
      address: parsed.data.address ?? "",
    };

    return { data, content: parsed.content };
  } catch (error) {
    console.error("Error reading contact.md:", error);
    return { data: null, content: null };
  }
}
```

Reply:
I will not commit any of the suggestions, as I wrote I have been working on a fix and committed it that fixes the typing and path issues.